motionplan/constraint.go

```diff
@@ -579,8 +579,12 @@ func ConstraintsFromProtobuf(pbConstraint *motionpb.Constraints) *Constraints {
 	orientConstraintFromProto := func(orientConstraints []*motionpb.OrientationConstraint) []OrientationConstraint {
 		toRet := make([]OrientationConstraint, 0, len(orientConstraints))
 		for _, orientConstraint := range orientConstraints {
+			orientTol := 0.
```

Comment:
why not set it to this https://github.com/viamrobotics/rdk/blob/main/motionplan/plannerOptions.go#L21


my reasoning is that 0 is very ambitious and might result in infeasibilty errors.

Reply:
this is happening, just in another section of the code.  This PR is just applying the same nil pointer dereference fix that Peter did here https://github.com/raybjork/rdk/commit/18a365f170c1e1fe903d944597609c77eded429d